### PR TITLE
Balance mail spawns and adds more mailboxes

### PR DESCRIPTION
**Pull request by cosmiccoincidence**
#### Summary
 ```SUMMARY: Balance "Balances mail spawns, and adds mailboxes to homes missing them"```  

#### Purpose of change
- Lowers the chance of magazines spawning compared to newspapers, flyers, and phone-books.
- Changed from 1 newspaper page spawning to multiple.
- Lowered minimum repeat for mail spawns from 1 to 0.

#### Additional context
removed line 9540 of item_groups.json which was a duplicate of "mag_comic" in the mail item group.

### CODE REVIEW

**Comment by Kelenius** [comment]
What's the reasoning behind this change?

**Comment by cosmiccoincidence** [comment]
> What's the reasoning behind this change?

Balance

I made the mail item group and mail boxes,  now I'm balancing them.

**Comment by kevingranade** [comment]
More specifically, you probably want:
```SUMMARY: Balance "Nerfs mail spawns"```
OR, if you want it treated as a minor tweak that doesn't appear in the changelog:
```SUMMARY: None```

**Comment by FulcrumA** [comment]
Just to check if I can do so in another's ticket:
Jenkins rebuild

**Comment by cosmiccoincidence** [comment]
> More specifically, you probably want:
> `SUMMARY: Balance "Nerfs mail spawns"`

Thanks, made the change.

**Comment by ZhilkinSerg** [comment]
Jenkins rebuild.

File: data/json/mapgen/house/house07.json [hunk 1]
```diff
@@ -89,7 +91,10 @@
         "r": "f_rack"
       },
       "toilets": { "t": {  } },
-      "items": { "Y": { "item": "coat_rack", "chance": 30, "repeat": [ 1, 4 ] } },
+      "items":{
```
Comment (ZhilkinSerg) [inline data/json/mapgen/house/house07.json]: Whitespace missing after colon.

File: data/json/mapgen/house/house_vacant.json [hunk 2]
```diff
@@ -1575,7 +1593,15 @@
...
-      "furniture": { "#": "f_null", ".": "f_null", "_": "f_null", "s": "f_null", "t": "f_null", "|": "f_null" },
+      "furniture": {
+        "#": "f_null",
+        ".": "f_null",
+        "_": "f_null",
+        "P": "f_mailbox",
+        "s": "f_null",
+        "t": "f_null",
+        "|": "f_null"
+      },
```
Comment (ZhilkinSerg) [inline data/json/mapgen/house/house_vacant.json]: Should be linted and look like earlier:

image

**Comment by cosmiccoincidence** [comment]
Still not perfect but a bit more balanced.

**Pull request merged**